### PR TITLE
Hosting the docs on RTD

### PR DESCRIPTION
**Pull request by keirthana**
This is a PR with all the changes required to host the docs on RTD. These changes are targeted for 1.23.
This is a draft PR as more changes will come in for the migration, however, reviews can happen in parallel.

Since this is a huge update (but the changes are mostly syntax-based and does not include technical changes), if reviewing the PR is difficult, you can [review the published content](https://canonical-anbox-cloud-documentation.readthedocs-hosted.com/en/latest/) which currently exists as a private project on RTD.

### CODE REVIEW

File: contribute/anbox-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## Images and screenshots
+
+While images and screenshots are very helpful in visual appeal and getting the message across to the readers, they can be hard to maintain and cause overhead if they change too often. So use images or screenshots only when they:
+
+* Do not change often requiring updates for every release.
+* Explain the concept better than text or it should strongly reinforce the concept explained in the text.
+* Add to the reader’s understanding rather than just providing a step by step visual of the user interface.
+* Break the monotony of continuous text information when trying to accomplish a tough procedure.
+
+Simple images can be made using an image editor of your choice or you can use [diagrams.net](https://www.diagrams.net).
```
Comment (ajanon) [inline contribute/anbox-style-guide.md:78]: (This is a bit unrelated to your changes)
I couldn't find anything specific for accessibility in the Ubuntu style guide.
In the context of images and screenshots, I see that it mentions alt-text. Do we have guidance on how to write alt-text for images for accessibility purposes?

File: explanation/production.md [hunk 2]
```diff
@@ -18,25 +21,25 @@ Consider the following questions to decide your networking requirements:
...
+* {ref}`exp-capacity-planning`
 
 ## Scalability, High availability, and redundancy
 
 With Anbox Cloud, you can scale your deployment to as many clusters and subclusters as you require. It is a good idea to define how many Anbox clusters you want to deploy and how many subclusters per cluster. 
 
 You should also assess the number of instances that you require for each model/service. For example, think about the number of Juju controller instances that you need. See [Make a controller highly available](https://juju.is/docs/olm/manage-controllers#heading--make-a-controller-highly-available) for more information.
 
-Anbox Cloud comes with support for High Availability (HA) for both Core and the Streaming Stack.You can define HA by adding new Juju units. See [How to enable High availability?](https://discourse.ubuntu.com/t/how-to-enable-high-availability/17754) to plan your HA requirements.
+Anbox Cloud comes with support for High Availability (HA) for both Core and the Streaming Stack.You can define HA by adding new Juju units. See {ref}`howto-enable-ha` to plan your HA requirements.
```
Comment (ajanon) [inline explanation/production.md:42]:
```suggestion
Anbox Cloud comes with support for High Availability (HA) for both Core and the Streaming Stack. You can define HA by adding new Juju units. See {ref}`howto-enable-ha` to plan your HA requirements.
```

File: explanation/instances.md [hunk 3]
```diff
@@ -71,27 +74,37 @@ An instance moves through different stages and correspondingly can have the foll
...
+* {ref}`howto-view-instance-logs`
+* {ref}`howto-wait-for-application`
+
+
+
+
+
+
+
+
```
Comment (ajanon) [inline explanation/instances.md:110]: As above, that's a lot of empty lines.

File: explanation/applications.md [hunk 4]
```diff
@@ -86,18 +92,23 @@ error_message: 'Failed to install application: com.foo.bar: Failed to extract na
...
+* {ref}`howto-delete-application`
+* {ref}`howto-list-applications`
+* {ref}`howto-scale-down-cluster`
+* {ref}`howto-test-application`
+* {ref}`howto-update-application`
+* {ref}`howto-wait-for-application`
+* {ref}`ref-application-manifest`
+
+
+
```
Comment (ajanon) [inline explanation/applications.md:114]: Empty lines here.

File: howto/install-appliance/azure.md
Comment (ajanon) [inline howto/install-appliance/azure.md:1]: Nearly everything in this file is indented, is that intended?

**Review by ajanon — CHANGES_REQUESTED** [review]
That's an impressive amount of changes! Thanks a lot for your great work!

I left a couple of comments inline.
I also have some questions:

There are a lot of files marked as 'orphan', what is this for?

I can see that a lot of the release-notes files have a first header of 'Introduction' instead of the version (see `1.20.0.md` for an example of this). As other files have the version as the title, I don't think it is intended.

**Comment by keirthana** [comment]
> That's an impressive amount of changes! Thanks a lot for your great work!
> 
> I left a couple of comments inline. I also have some questions:
> 
> There are a lot of files marked as 'orphan', what is this for?

That indicates pages that do not belong in the TOC navigation. For example, the individual release notes won't be available from the TOC as they would cause too much clutter but are tied together using the release notes landing page.

> 
> I can see that a lot of the release-notes files have a first header of 'Introduction' instead of the version (see `1.20.0.md` for an example of this). As other files have the version as the title, I don't think it is intended.

Thanks for catching this, this is a by-product of using discourse posts as release announcements. I have fixed them now but haven't really bothered to standardise the legacy release notes format with the release date included.

File: .custom_wordlist.txt
[the remark this answers is not in the document]
Comment (keirthana) [reply .custom_wordlist.txt:1]: I have removed these words from the list but will be working on automating this check. I would like to see if I can contribute this to the parent [documentation-workflows](https://github.com/canonical/documentation-workflows/) checks from where we derive our checks, so this may take a bit of time.

File: contribute/anbox-style-guide.md (hunk 1 above)
Comment (keirthana) [reply contribute/anbox-style-guide.md:78]: Some good resources I collected for now: https://webaim.org/techniques/alttext/ and https://design-system.service.gov.uk/styles/images/#alt-text

I will select appropriate guidelines and add this into our Anbox style guide but it will have to wait until later

**Comment by keirthana** [comment]
> > > I can see that a lot of the release-notes files have a first header of 'Introduction' instead of the version (see `1.20.0.md` for an example of this). As other files have the version as the title, I don't think it is intended.
> > 
> > 
> > Thanks for catching this, this is a by-product of using discourse posts as release announcements. I have fixed them now but haven't really bothered to standardise the legacy release notes format with the release date included.
> 
> Will you push these changes to your branch and make them part of this MP, or will this come in a later step? I guess this is only important for the latest release notes (1.22.0, maybe 1.21.X too?)

They are part of this PR now. As discussed in the SU, I will not be merging this and making this our main branch while the current main branch will become discourse-docs

File: .custom_wordlist.txt
[the remark this answers is not in the document]
Comment (keirthana) [reply .custom_wordlist.txt:1]: When I ran this in my local branch, it returns `NVIDIA` as well in spite of it being used in multiple places. Also, `Keystore` and `Biometrics` are being used. I am guessing there is some case-sensitive issue going on in here. 

Anyway, I am going to separate this effort.  This is a good idea for automated checks, it just needs some focused effort.

File: contribute/anbox-style-guide.md (hunk 1 above)
Comment (ajanon) [reply contribute/anbox-style-guide.md:78]: Thanks for the links!
